gnomad_hail/utils/relatedness.py

```diff
@@ -0,0 +1,341 @@
+import hail as hl
+from gnomad_hail import logger
```

Comment:
we've been using this strategy throughout the library, so that all logging goes through the same logger. That said, I've definitely started feeling that this strategy  was coming to its limits and I'm not opposed to reconsidering. What I'll do is add an issue as this is a more general question for the library.